fips202/fips202.h

```diff
@@ -35,6 +35,23 @@ typedef struct
     uint64_t ctx[25];
 } shake256ctx;
 
+/**
+ * Squeezes blocks of data from the Keccak state.
+ *
+ * This function extracts data from the Keccak state, effectively performing the squeeze step of the Keccak
+ * cryptographic hash function. It generates output data based on the current state and updates the state
+ * accordingly.
+ *
+ * @param h Pointer to the output buffer where the squeezed data will be stored.
+ * @param nblocks Number of blocks to squeeze from the state. Each block is r bytes long, where r is the rate
+ *        specified when initializing the state.
+ * @param s Pointer to the Keccak state array. This array must contain at least 25 elements of type uint64_t.
+ * @param r Rate in bytes. Specifies the rate parameter of the Keccak function, which determines how much
+ *        data is processed per permutation.
+ */
+void keccak_squeezeblocks(uint8_t *h, size_t nblocks, uint64_t *s, uint32_t r);
+
+
```

Comment:
Why did you have to expose this function? I don't think it should be exposed. 
You should only be using the `shake128_squeezeblocks` and `shake256_squeezeblocks` functions

Reply:
I have to expose this function to switch to single lane Keccak. 
As in `gen_matrix`, when the `rej_sampling` fails to fullfill a vector, additional for a single lane is executed.
So a single lane state data is feed into `keccak_squeezeblocks` function. 
I'll see if I can wrap this function in another way.

Reply:
okay, I add a function call `shake256x1_squeezeblocks` replace the direct call to `keccak_squeezeblocks`.